https://gitlab.freedesktop.org/plymouth/plymouth/-/commit/5f1e43c00039a7fe1fff768b91a05a695fb4a53d
https://bugs.gentoo.org/863839

From 5f1e43c00039a7fe1fff768b91a05a695fb4a53d Mon Sep 17 00:00:00 2001
From: Ray Strode <rstrode@redhat.com>
Date: Wed, 3 Aug 2022 15:23:33 -0400
Subject: [PATCH] ply-utils: Drop linux/fs.h include

It was needed long ago for a function we no longer even have.

Now it's causing compile errors on Fedora 37 because it's conflicting
with sys/mount.h.

This commit drops it.
--- a/src/libply/ply-utils.c
+++ b/src/libply/ply-utils.c
@@ -46,7 +46,6 @@
 #include <sys/user.h>
 #include <sys/wait.h>
 #include <time.h>
-#include <linux/fs.h>
 #include <linux/vt.h>
 
 #include <dlfcn.h>
GitLab
